js_formatter.py

```diff
@@ -34,20 +32,13 @@ def _try_get_short_path(path):
 # crazyness to get jsbeautifier.unpackers to actually import
 # with sublime's weird hackery of the path and module loading
 add_lib_path(libs_path)
-add_lib_path(os.path.join(libs_path, "six-1.8.0"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier", "jsbeautifier"))
```

Comment:
@mathieug I remember a lot of pain early on in the project getting this stuff to work, and a lot of explicit includes seemed necessary. I am by no means good at python, but have you tested this pretty thoroughly across sublime versions and operating systems?